### ISSUE TITLE
Guitar Sniffer not recognizing button inputs

### ISSUE THREAD

**Issue by BlackMemeChef** [artman41/guitarsniffer#15]
I know there is already an issue open of this same problem, but it's old and I'm having the same problem, so I figured I'd make a new one.

Basically, Guitar Sniffer isn't recognizing any button inputs from my guitar. I'm using an XB1 Stratocaster with the Xbox wireless adapter. I've got WinPCap and USBPCap all installed, and just nothing works. I've uninstalled and reinstalled everything several times, and I've tried NPCap instead of WinPCap, but nothing has worked.

Everything used to work fine and without any problems. I just was trying to play Clone Hero today and found that nothing was working.

Edit: Forgot to mention that I have vJoy installed too, and have it set up correctly (at least to my knowledge).

**Comment by Dunkalunk** [comment]
If you've tried everything in all of the comments in that other issue posted [here](https://github.com/artman41/guitarsniffer/issues/12) then unfortunately I have nothing else to offer.  I was hoping someone having the issue would find what was wrong and report it on that issue thread but so far nothing.  Sorry :(

**Comment by BlackMemeChef** [comment]
Yeah, I checked that post and nothing in there seemed to work for me.

It's just frustrating because it had been working perfectly fine, and then just randomly stopped. Nothing changed that should've made it stop working.

I appreciate your comment though. Hopefully someone can figure this out.

**Comment by Dunkalunk** [comment]
Yeah I remember a while back after a Windows update or something that mine stopped working too, but uninstalling and reinstalling WinPCap worked for me and I haven't had any trouble since.  I still would like to implement logging into the project (even though it's not my project) so we can potentially see what's going on.  Just haven't had the time in so long...

**Comment by sorcer0r** [comment]
Make sure to uninstall Npcap - it does not work if you have it installed.

**Comment by Rhystic1** [comment]
Unfortunately it's not working for me anymore either.

Tried reinstalling everything, but no matter what I try GuitarSniffer starts, but it does not detect any input whatsoever.
I'm on Windows 11.

**Comment by sorcer0r** [comment]
With windows 11 only the old sniffer works:
https://github.com/artman41/FenderStratocastorSniffer
